app-misc/mc/mc-4.8.33.ebuild

```diff
@@ -99,18 +92,7 @@ src_configure() {
 }
 
 src_test() {
-	# Bug #759466
-	if [[ ${EUID} == 0 ]] ; then
-		ewarn "You are emerging ${PN} as root with 'userpriv' disabled."
-		ewarn "Expect some test failures, or emerge with 'FEATURES=userpriv'!"
-	fi
```

Comment:
Please leave this `if` in, we still need the warning.

Reply:
Yes, I will revert this for now, because, unfortunately, there are still problems left with root, but when/if we fix the test suite this can really be removed, correct?

Reply:
I've added a note that this is currently being tracked upstream.